src/modules/internet/index.ts

```diff
@@ -155,6 +159,73 @@ export class InternetModule {
         ])}${lastName}${this.faker.number.int(99)}`;
         break;
     }
+
+    //There may still be non-ascii characters in the result.
+    //First remove simple accents etc
+    result = result
+      .normalize('NFKD') //for example è decomposes to as e +  ̀
+      .replace(/[\u0300-\u036f]/g, ''); // removes combining marks
+
+    result = result
+      .split('')
+      .map(function (char) {
```

Comment:
Please use inline lambdas, not `function (char) {`

```suggestion
      .map((char) => {
```

Not sure if you need to reformat after apply suggestion